dcicutils/misc_utils.py

```diff
@@ -464,6 +464,33 @@ def as_datetime(dt, tz=None):
         return None
 
 
+def as_ref_datetime(dt):
+    """
+    Parses a given datetime, returning a rendition of that tie in the reference timezone (US/Eastern by default).
+
+    If the input time is a string or a naive datetime with no timezon, it is assumed to be in the reference timezone
+    (which is US/Eastern by default).
+    If the time is already a datetime, no parsing occurs, but the time is still adjusted to present as it would in the
+    reference timeszone.
+    """
+    dt = as_datetime(dt, tz=REF_TZ)
+    hms_dt = dt.astimezone(REF_TZ)
```

Comment:
Isn't it possible for `as_datetime` to return None, hence throwing `AttributeError` below? Or do you expect that will never happen with this function? Same question in the next function.

Reply:
Good catch. It may not come up but should behave predictably there.

Reply:
Fixed. Added unit testing.